### ISSUE TITLE
Engage

### ISSUE THREAD

**Issue by Maj2004** [EasyFarm/EasyFarm#409]
I cant get it to engage it just runs up to a mob and does nothing

**Comment by Maj2004** [comment]
I dont get why its not engaging i have tried everything....

**Comment by Maj2004** [comment]
Yeah i think i fixed that issue, the mobs its farming do not aggro at all I believe i have everything figured out with it now, except for the issue it just loves to crash randomly lol.. i assume its something im doing wrong but i dont know... thanks for the replies.

**Comment by Yagamyi** [comment]
My question is how to i add ws, magic, ja's, etc without the program closing and shutting down on me? Im using windower could that be an issue? Do i have to run ffxi in full screen to have bot work? 

Bc right now it just freezes and exits without any notification as soon as i add a ws, magic, ja, anything.  

Thanks in advance!

**Comment by Vobant** [comment]
I run mine in window mode and use ashita. I had not used windower in a while not sure if thats the issue or not. So I am not sure.


________________________________

**Comment by Jinko1** [comment]
I noticed it crashed on Ashita more than on Windower and make sure you are pausing it whilst you edit or add moves as running the bot whilst editing abilities and spells can make it crash for some reason.

**Comment by ShadowrunQuetzalcoatl** [comment]
Never crashes on ashita for me running 11 at a time, unless i dont use the autofill option. and you dont have to pause it to add stuff just dont enable it before doing auto fill.
